app/views/include/edit_button.volt

```diff
@@ -0,0 +1,3 @@
+<a href="https://github.com/phalcon/docs/blob/3.2/{{ language }}/{{ page }}.md" class="" style="position: fixed; bottom: 16px; right: 24px; z-index: 100;">
```

Comment:
Could we use `{{ version }}` or something like that instead of `3.2`?

Reply:
I would like to see all the styles in the common place, e.g. `style.css`

Reply:
Also I'm pretty sure we need media queries here to adjust button size for phones.

Reply:
No problem, I will take care of these as soon as I get a chance